coralproject/talk
# allow `utilities/download` to work with archive databases

## What does this PR do?

Updates the `utilties/download` so it can work with archive databases.

Will now use tenantID if the tenant is not available (as there is no `tenants` collection inside archive databases)

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- create an archive mongo database (or pull one from Atlas)
- set up the config as per the readme for this connection
- download a tenant (specified in your config)
- see that it creates the output folder using the `tenantID` instead of the org name
- see that the archived comment data is downloaded appropriately

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.
